docs/protocol-methods.rst

```diff
@@ -552,6 +549,117 @@ Protocol version 1.0 returning an error as the result:
+  that the backing bitcoind supports relay of transaction packages. For example, note that Bitcoin Core 26.0
+  already exposes the `submitpackage` RPC however it is effectively non-functional until Bitcoin Core 28.0.
+
+**Result Example**
+
+When *verbose* is :const:`false`:
+
+    Successful broadcast::
+
+        {
+          "success": true
+        }
+
+    With errors::
+
+        {
+          "success": false,
+          "errors":
+          [
+            {
+              "txid": "ec6f295cd4b1b91f59cabb0ab8fdc7c76580db08be6426e465f75a69d82b9659",
+              "error": "bad-txns-inputs-missingorspent"
+            }
+          ]
+        }
+
+When *verbose* is :const:`true` (exact structure depends on bitcoind impl and version, and should not be relied upon)::
+
+    {                                   (json object)
+      "package_msg" : "str",            (string) The transaction package result message. "success" indicates all transactions were accepted into or are already in the mempool.
+      "tx-results" : {                  (json object) transaction results keyed by wtxid
+        "wtxid" : {                     (json object) transaction wtxid
+          "txid" : "hex",               (string) The transaction hash in hex
+          "other-wtxid" : "hex",        (string, optional) The wtxid of a different transaction with the same txid but different witness found in the mempool. This means the submitted transaction was ignored.
+          "vsize" : n,                  (numeric, optional) Sigops-adjusted virtual transaction size.
+          "fees" : {                    (json object, optional) Transaction fees
+            "base" : n,                 (numeric) transaction fee in BTC
+            "effective-feerate" : n,    (numeric, optional) if the transaction was not already in the mempool, the effective feerate in BTC per KvB. For example, the package feerate and/or feerate with modified fees from prioritisetransaction.
+            "effective-includes" : [    (json array, optional) if effective-feerate is provided, the wtxids of the transactions whose fees and vsizes are included in effective-feerate.
+              "hex",                    (string) transaction wtxid in hex
+              ...
+            ]
+          },
+          "error" : "str"               (string, optional) The transaction error string, if it was rejected by the mempool
+        },
+        ...
+      },
+      "replaced-transactions" : [       (json array, optional) List of txids of replaced transactions
+        "hex",                          (string) The transaction id
+        ...
+      ]
+    }
+
+.. note:: If the input is malformed, notably if `raw_txs` is not a well-formed package,
+  errors will be sent back as JSON RPC ERROR, as usual.
+  Ultimately the distinction can depend on bitcoind.
+  Compare examples (id=3 gets JSON RPC ERROR, but id=4 gets success=false with errors array)::
```

Comment:
After implementing this RPC in electrum (client), I think this note (and examples) should be here for other implementers to see. It is not immediately obvious that errors can come back at two different abstraction layers.